CommunityToolShedMvc/Controllers/ToolController.cs

```diff
@@ -70,5 +71,45 @@ public ActionResult Create(FormCollection collection, int communityid)
             return RedirectToAction("Index", "Community", new { id = communityid });
         }
         
+        public ActionResult Borrow(int communityid, int id)
```

Comment:
This action method is changing data on the server, so decorate it with an `HttpPost` attribute.